### ISSUE TITLE
Job Card with sub operation issue (operation link is read only & error in total completed qty  )

### ISSUE THREAD

**Issue by bhargav007n** [frappe/erpnext#38939]
### Information about bug

Basically we have two issues in job card with operations having sub operations.
1. Operation link in the time log is read only. We can not select the sub operation link.
2. Error calculation in total  completed qty for job card with sub operation.

Actual calculation should be like below-
lets take an example of job card with 6 sub operation and with 125 no's as Qty to manufacture.
        Sub operation   ||   Completed Qty
               (name)        ||          (no's)
                  1              ||           125 no's
                  2              ||          124 no's
                  3              ||          124 no's
                  4              ||          123 no's                            
                  5              ||          122 no's
                  6              ||          121 no's

So the total completed qty should be **121 no's** . The logic behind this is which ever the least qty in all of the sub operation that should be the **Total completed qty**.
Secondly, Process loss should be difference of Qty to manufacture and completed qty .
  i.e **Process loss= Qty to manufacture - completed qty** 
[0347-BFTH grinding - PO-JOB00209.pdf](https://github.com/frappe/erpnext/files/13761843/0347-BFTH.grinding.-.PO-JOB00209.pdf)

### Module

manufacturing

### Version

ERPNext: v14.48.1 (version-14)

Frappe Framework: v14.55.0 (version-14)

Frappe HR: v14.17.0 (version-14)

India Compliance: v14.19.4 (version-14)

Payments: v0.0.1 (develop)

### Installation method

manual install

**Comment by rohitwaghchaure** [comment]
Fixed https://github.com/frappe/erpnext/pull/49255
